src/components/Questions.tsx

```diff
@@ -0,0 +1,55 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronDown, faChevronUp } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+import GetPlanButton from "./GetPlanButton";
+import React from "react";
+import { QUESTIONS } from "constants/constants";
+
+interface IQuestionListProps {
+  question?: string;
+  answer?: string;
+  key?: number;
+}
+
+interface IQuestionsProps {}
+
+const QuestionsList: React.FC<IQuestionListProps> = ({ question, answer }) => {
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
it's ok to pass a type in useState, but in this case TS knows and will accept only boolean if you will pass a boolean as initial state. it's not a mistake or smth, but it's not very efficient